/**/ron_configuration:
    ros__parameters:
        robot_node_bottom_face_translation: -0.05
        robot_node_top_face_translation: 0.05
